rubocop/README.md

```diff
@@ -0,0 +1,36 @@
+# How to use Rubocop configs
```

Comment:
re: @caffeineandbytes's comment, I don't see any reason not to rename this directory to `ruby` instead of `rubocop`.

Reply:
@caffeineandbytes and @bradlamotte are you thinking the file structure would be `ruby/rubocop/file_name.yml` or `ruby/file_name.yml`? rubocop will have a lot of files within it, but I'm not sure if there are any other ruby style guides that will be in here as well.

Reply:
Personally, I was thinking just `ruby/filename.yml`